it/controllers/IncomeBeforeTaxPayeItSpec.scala

```diff
@@ -67,7 +80,7 @@ class IncomeBeforeTaxPayeItSpec extends IntegrationSpec {
 
         val result = route(fakeApplication(), request)
 
-        result.map(status) mustBe Some(SEE_OTHER)
+        result.map(status) mustBe Some(INTERNAL_SERVER_ERROR)
```

Comment:
description of the test does not match the expected value. Is the change from see_other to internal_server_error intentional?

Reply:
Whoops, forgot the name. Before the HttpVerbs change an error would return an okay or it would redirect, but now any case that isn't an AtsNotFoundResponse return a 500 instead

Fixed